pom.xml

```diff
@@ -1461,7 +1461,7 @@
                             <plugin>
                                 <groupId>org.codehaus.mojo</groupId>
                                 <artifactId>findbugs-maven-plugin</artifactId>
-                                <version>2.4.0</version>
+                                <version>3.0.5-SNAPSHOT</version>
```

Comment:
Do we really have to depend on the `SNAPSHOT`? Would prefer to depend on real releases.